dags/hivemind_etl_helpers/tests/integration/test_github_etl_fetch_commits.py

```diff
@@ -1,25 +1,28 @@
 from datetime import datetime
 from unittest import TestCase
 
+from dags.hivemind_etl_helpers.src.db.github.extract import GithubExtraction
 from github.neo4j_storage.neo4j_connection import Neo4jConnection
-from hivemind_etl_helpers.src.db.github.extract import fetch_commits
 
 
 class TestFetchCommits(TestCase):
     def setUp(self) -> None:
         neo4j_connection = Neo4jConnection()
         self.neo4j_driver = neo4j_connection.connect_neo4j()
+        self.github_commit_extractor = GithubExtraction()
         with self.neo4j_driver.session() as session:
             session.execute_write(lambda tx: tx.run("MATCH (n) DETACH DELETE (n)"))
 
     def test_get_empty_results_no_from_date(self):
         repository_ids = [123]
-        commits = fetch_commits(repository_id=repository_ids, from_date=None)
+        commits = self.github_commit_extractor.fetch_commits(
```

Comment:
Please name the `github_commit_extractor` to `github_extractor`, as our goal in the future is for all other extraction parts (issue, comment, pull request) of github, we want the `GitHubExtraction` class to inherit from each of those extractors.